src/forms/spaceEditSchema.ts

```diff
@@ -19,12 +19,22 @@ const roomImageUrlSchema = Yup.string().required(
   `${ROOM_TAXON.capital} icon is required`
 );
 
+const roomSchema = Yup.object().shape<RoomSchemaShape>({
```

Comment:
One schema per file. Makes it easier to discover them and mix and match them

Reply:
split 2 new schemas into 2 separate files c72f598